README.md

```diff
@@ -30,6 +30,13 @@ Update all dependencies:
 ```shell
 helm dependency update charts/activiti-cloud-full-example
 ```
+Create Activiti Keycloak Client Kubernetes secret in the current namespace:
```

Comment:
I like this but just have one concern, is it ok to do it here or would it be better to have a separate PR as I believe this is not the only sensitive value which would benefit from such a treatment and at least we should have a follow up PR that "completes" this also for other password/secrets we have

Reply:
IMO, it's fine to have it here, as it's handling new properties introduced by this PR and did not exist before. However, it makes sense to create another PR to handle other sensitive values in a similar way.